styles/blocks/about.scss

```diff
@@ -0,0 +1,30 @@
+@use '../utils/mixins.scss' as *;
+
+.about {
+
+  @include grid();
+
+  &__title {
+    grid-column: span 2;
+
+    @include for-mobile-big() {
+      grid-column: span 3;
+    }
+
+    @include for-desktop() {
+      grid-column: span 5;
+    }
+  }
+
+  &__benefits {
+    grid-column: span 2;
+
+    @include for-mobile-big() {
+      grid-column: span 3;
+    }
+
+    @include for-desktop() {
+      grid-column: span 5;
+    }
+  }
+}
\ No newline at end of file
```

Comment:
It's a good practice to end your file with a newline. This is a standard convention that some tools might rely on, to recognize or process the file correctly.